src/Uno.Toolkit.UI/Extensions/DispatcherQueueExtensions.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.System;
+
+namespace Uno.Toolkit.UI
+{
+	internal static class DispatcherQueueExtensions
```

Comment:
wouldnt it be better to expose this from DependencyObject, so you can drop that ugly `_dispatcherQueue`

Reply:
The _dispatcherQueue field is required so that we can capture the DispatcherQueue when we know that it exists (ie when the control gets created on the UI thread)

Reply:
Good catch - that should be assignment to field instead of defining a property!